src/components/Account.tsx

```diff
@@ -9,7 +9,7 @@ const AccountComponent: React.FC<AccountComponentProps> = ({
   address,
   disconnect,
   ensName,
-  ensAvatar,
+  ensAvatar, // why are there unused imports here?
```

Comment:
I remember removing unused imports and anything else during the build process. Eslint was on. My bad!